### PR TITLE
Fixes cult forever

### PR DESCRIPTION
**Pull request by ShiftyRail**
I expect the winrates to go as high as 66.6%

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
what about their lawset

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm [hunk 1]
```diff
@@ -827,7 +832,7 @@
 		activator.client.images |= progbar
 
 	//secondly, let's stun our victim and begin the ritual
-	to_chat(victim, "<span class='danger'>Occult energies surge from below your feet and seep into your body.</span>")
+	to_chat(victim, "<span class='danger'>Occult energies surge from below your [issilicon(victim) ? "actuators" : "feet"] and seep into your [issilicon(victim) ? "chassis" : "body"].</span>")
```
Comment (SonixApache) [inline code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:835]: ableist towards borgs with no actuator and humans with no feet

**Comment by beesil** [comment]
oh god, oh fuck

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:835]: They're metaphorical actuators you bigot

**Comment by MadmanMartian** [comment]
You'll want to unsync them from the AI, too, lest they just be restored the next time the AI checks their laws.

**Comment by iFlashYou** [comment]
based and cultpilled

**Comment by Sooose** [comment]
Shifty making the big play by making this an April fools PR so it can't be added as an actual feature later.

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm [hunk 2]
```diff
@@ -1013,6 +1018,14 @@
 				abort(RITUALABORT_CONVERT)
 				message_admins("BLOODCULT: [key_name(victim)] has been converted by [key_name(converter)].")
 				log_admin("BLOODCULT: [key_name(victim)] has been converted by [key_name(converter)].")
+				if (issilicon(victim))
+					var/mob/living/silicon/S = victim
+					S.laws = new /datum/ai_laws/cultimov
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:1023]: Isn't there a function to set the laws?

Comment (ShiftyRail) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:1023]: Not that I saw, I think it's handled this way in other places (notably emagging)

**Comment by Intigracy** [comment]
neat

**Comment by DeityLink** [comment]
>Shifty wants this bad to watch the silicon be subverted by the cult so they can no longer be considered cult enemies.

I'm unironically hoping this remains post-April Fools to observe this destroy the meta.

**Pull request merged**